<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeDataSet</class>
 <widget class="QWidget" name="VolumeDataSet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>233</width>
    <height>230</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>233</width>
    <height>230</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>233</width>
    <height>230</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>三维数据分层</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>96</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Center</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>127</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Bottom</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>155</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Show</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_AllNumber">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>15</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>All</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_7">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>201</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="QPushButton" name="m_pbSave">
    <property name="text">
     <string>应用</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="text">
     <string>应用退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>66</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Top</string>
   </property>
  </widget>
  <widget class="QSplitter" name="splitter_4">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>60</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <widget class="QSplitter" name="splitter_5">
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QSplitter" name="splitter_6">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSpinBox" name="m_topS"/>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>-----&gt;</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="m_topE"/>
     </widget>
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSpinBox" name="m_centerS"/>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>-----&gt;</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="m_centerE"/>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSpinBox" name="m_bottomS"/>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>-----&gt;</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="m_bottomE"/>
     </widget>
     <widget class="QComboBox" name="m_cbSelect">
      <item>
       <property name="text">
        <string>Top</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Center</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Bottom</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
